### DISCUSSION TITLE
Any way to adjust narrator's piercing voice?

### DISCUSSION THREAD

**Discussion by crackpaperscissors** [yermak/AudioBookConverter#456] [General]
Howdy. Not sure if this is the correct place to ask this. But sometimes the narrator's voice is too high pitched. It makes listening to the audiobook unbearable. Any settings in the software I can adjust to help with that?

**Comment by yermak** [comment]
Currently software does not support this.
You can experiment with cut-off parameter as it will cut frequencies higher then parameter set.
I may look into this in the future, if there are more people interested in it. I'm only concerned that output quality will be very subjective, and there is no good way to define what is "good output".
